src/utils/clone-deep.js

```diff
@@ -1,5 +1,44 @@
-function cloneDeep(obj) {
-  return JSON.parse(JSON.stringify(obj));
+function isObject(value) {
+  return Object.prototype.toString.call(value) === '[object Object]';
+}
+
+function isArray(value) {
+  return value instanceof Array;
```

Comment:
Better: `Array.isArray(value)`

Reply:
Yes of course, thank you, changed.

Would have been nice to have this on `Date` as well 😊